node/server_functions.js

```diff
@@ -3266,27 +3266,34 @@ function appengine_call(method, args, on_success, on_error) {
 }
 
 function discord_call(message) {
+	// TODO: should it not post to the channel either way? I vote to remove this return
```

Comment:
I'd argue that the new `variables.DISCORD.ENABLED` configuration option has indeed made this check obsolete.

Reply:
I think standard, hardcore and test servers are run from the same directory though. So they all share the same variables.js file.

Should the test server actually post to the same channels as the live servers? The same question is true for hardcore servers.

I kinda feel they perhaps need their own configuration as theese two servers are  a special kind of servers.